lightController/lightController.ino

```diff
@@ -144,6 +154,71 @@ void draw_rainbow()
     }
 }
 
+
+void vibe_sparkle() 
+{
+   if(is_button_pressed) 
+   {
+    for(int i = 0; i < NUM_LEDS; i++)
+    {
+	uint8_t bri = 255;
+		leds[i] = CHSV(global_hue, 200, bri);
+    addGlitter(10);
+	    }
+    }
+    else {
+      for(int i = 0; i < NUM_LEDS; i++){
+      leds[i] = CHSV(0, 0, 0);
+      } //If no button press, LEDs off
+    }
+  
+  //Color change if button is released
+  if(!is_button_pressed & prev_state) {
```

Comment:
This does not do what you think it does.

use `!is_button_pressed && prev_state`